opentelekomcloud/services/ces/resource_opentelekomcloud_ces_alarmrule.go

```diff
@@ -371,14 +350,16 @@ func resourceAlarmRuleRead(ctx context.Context, d *schema.ResourceData, meta int
 		return fmterr.Errorf(errCreationClient, err)
 	}
 
-	r, err := alarmrule.Get(client, d.Id()).Extract()
+	r, err := alarms.ShowAlarm(client, d.Id())
```

Comment:
`r` is not a very good name for the variable.

Reply:
SDK: Also, strange that `showAlarm` which is accepting unique ID returns list.

Reply:
Well that's problems with API itself... I will change the name

Reply:
Anyway, that doesn't relate to this PR :)

Reply:
I will change this name.

Reply:
Ok, I will not change this name but change the return value.
It returns a list, but it always has at most one value.

Reply:
@Aloento There is a question though - if there is no values, should it return `nil, nil`, which is regarded as bad practice (and even a error for some linters), or standard SDK 404 error. I would suggest returning slice in case API returning slice 🤷 If I remember correctly, there are such cases already in the SDK.